notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-989_analysis_inertia_compensation.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        ax.set_xlabel("Time [UTC]")
</pre><p>is there a reason the hp plot has an xlabel, even though all the tickmark labels are on the velocity plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>